.travis.yml

```diff
@@ -1,5 +1,7 @@
 ---
 language: ruby
+dist: bionic
+os: linux
```

Comment:
Is the os necessary?

Reply:
No, it's just an info message that it's missing `[info] on root: missing os, using the default "linux"`